cmd/openshift-install/create.go

```diff
@@ -141,7 +151,7 @@ var (
 		assets: targetassets.Cluster,
 	}
 
-	targets = []target{installConfigTarget, manifestsTarget, ignitionConfigsTarget, clusterTarget}
+	targets = []target{installConfigTarget, manifestsTarget, ignitionConfigsTarget, clusterTarget, singleNodeignitionConfigsTarget}
```

Comment:
I believe `singleNodeignitionConfigTarget` needs to match what was used on line 82?